pyspline/utils.py

```diff
@@ -402,7 +402,7 @@ def tfi2d(e0, e1, e2, e3):
 
 
 def line_plane(ia, vc, p0, v1, v2):
-    r"""
+    """
```

Comment:
Hm what's the reason you removed the raw string? I added it there so that the LaTeX math backslashes don't trigger flake8 warnings -- the alternative is to use double backslashes everywhere and that's not as readable

Reply:
I thought it was a typo I made, I'll put it back in